project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,149 @@
+and in the [US by the ACLU and EFF]. The reasons outlined are exactly why the
+collaborators working on AnonCreds continue to evolve the capabilities.
+
+[Europe about privacy being at the heart of the European digital identity]: https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/issues/66
+[US by the ACLU and EFF]: https://www.eff.org/document/10-16-2023-aclu-eff-epic-comments-re-tsa-nprm-mdls
+
+The [AnonCreds Quarterly Activity Dashboard] shows the activity on the project
+repositories, with 97 commits by 12 contributors in the quarter. Those counts are
+each up by about 50% from last quarter.
+
+[AnonCreds Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Fanoncreds/dashboard;subTab=technical?time=%7B%22from%22:%222023-07-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-09-30T07:00:00.000Z%22%7D
+
+## Questions/Issues for the TOC
+
+None.
+
+## Releases
+
+- [anoncreds-rs] -- v0.2.0-dev.1
+- [anoncreds-clsignatures-rs] — v0.1.0, v0.2.0, v0.2.1, v0.2.2, v0.2.3
+
+## Overall Activity in the Past Quarter
+
+Coding for the quarter focused on enabling the use of the [AnonCreds Rust] in
+the various Aries frameworks, including [Aries VCX], [Aries Framework
+JavaScript] and [Aries Bifold]. Work is continuing on adding support in [Aries
+Cloud Agent Python] for the new library. Much of the coding in the [AnonCreds Rust] library
+itself was on integrating the [AnonCreds CL Signatures Rust] library, and on the
+wrappers that enabled use in the Aries frameworks.
+
+The [Hyperledger Mentorship program] enabled the creation of an [AnonCreds
+Mentorship project], that resulted in our bringing on [Aritra Bhaduri] who has
+been extremely effective at updating the [AnonCreds Specification] to include
+details of the cryptographic operations occurring during the issuing, receiving,
+presenting, and verifying (with revocation) of AnonCreds verifiable credentials
+and presentations. This gets us through the last major hurdle in completing the
+AnonCreds v1 specification. Remaining is an editorial pass through the entire
+specification for cleanup and consistency -- the easy stuff! And, since all of the
+objects and operations will remain in AnonCreds v2, the v1 specification is a
+huge head start towards the creation the v2 specification. Huge thanks and
+appreciation of the efforts by Aritra who has done an outstanding job!
+
+[Hyperledger Mentorship program]: https://wiki.hyperledger.org/display/INTERN/Hyperledger+Mentorship+Program
+[Aritra Bhaduri]: https://github.com/aritroCoder
+[AnonCreds Mentorship]: https://wiki.hyperledger.org/display/INTERN/Document%2C+Review%2C+and+Implement+Hyperledger+AnonCreds+ZKP+Cryptographic+Primitives
+
+The contribution of the new AnonCreds v2 code base, and underlying cryptography
+in [Hyperledger Labs Agora] represents a big opportunity to advance AnonCreds
```

Comment:
```suggestion
in [Agora], a Hyperledger Lab represents a big opportunity to advance AnonCreds
```

Reply:
In line with "Labs Representation" of the [Lab Proposal](https://docs.google.com/document/d/1VbgmlIqDnUnvjxSJ5QTUBV-Qn3eA7Gduw0edjYgDXIY/edit)

Reply:
Updated.